cmp/grid/GridModel.js

```diff
@@ -1085,9 +1103,26 @@ export class GridModel {
         };
     }
 
+    parseChooserModel(chooserModel) {
+        if (XH.isMobileApp) {
+            return chooserModel ? this.markManaged(new MobileColChooserModel(this)) : null;
+        }
+
+        if (chooserModel instanceof ColChooserModel) {
+            return new DesktopColChooserModel(chooserModel);
+        }
+
+        if (isPlainObject(chooserModel)) {
+            return this.markManaged(new DesktopColChooserModel(defaults(chooserModel,
+                {gridModel: this})));
```

Comment:
Would you mind fixing up the whitespace/line nesting here?  This is a bit of an odd line break.